include/dlaf/matrix/matrix.h

```diff
@@ -138,7 +163,7 @@ class Matrix : public Matrix<const T, D> {
   /// All accesses to the sub-pipelined matrix are sequenced after previous accesses and before later
   /// accesses to the original matrix, independently of when tiles are accessed in the sub-pipelined
   /// matrix.
-  Matrix subPipeline() {
+  Matrix subPipeline() noexcept {
```

Comment:
I don't mind, but I just want to clarify that the pipelines (and vector of pipelinses) also allocate, so this can throw.

Reply:
We're leaving it like this.